h-raylib.cabal

```diff
@@ -58,10 +63,13 @@ library
   default-language:   Haskell2010
 
   if flag(platform-windows) || (flag(detect-platform) && os(windows))
-    extra-libraries:  opengl32 gdi32 winmm shell32
+    if flag(mingw-cross)
+      extra-libraries:  opengl32 gdi32 winmm shell32 gcc
```

Comment:
When we are cross-compiling, we need to declare the `gcc` dependency since we do in fact depend on those mingw system libraries (such as `mcfgthread-12.dll`). This is only relevant when we are cross-compiling to Windows, so I added an extra flag for it.